### PR TITLE
Assume a denial if permissions have not been loaded yet

### LINKED ISSUE #4716: Error Sources/Security.php
Error : array_intersect(): Argument 2 is not an array

Sources/Security.php: 918
if (count(array_intersect($permission, $user_info['permissions'])) != 0)

Solution:

Sources/Security.php: 905
if (empty($user_info) || !isset($user_info['permissions']))

### PR DESCRIPTION
**Pull request by live627**
Fix #4716

### CODE REVIEW

File: Sources/Security.php [hunk 1]
```diff
@@ -906,6 +906,11 @@ function allowedTo($permission, $boards = null, $any = false)
 	if (empty($user_info))
```
Comment (Sesquipedalian) [inline Sources/Security.php]: Why not just add ` || !isset($user_info['permissions'])` to this line instead?

**Review by Sesquipedalian — CHANGES_REQUESTED** [review]

File: Sources/Security.php (hunk 1 above)
Comment (live627) [reply Sources/Security.php]: Then I'd lose the comment.

Comment (Sesquipedalian) [reply Sources/Security.php]: 😆 Well, I suppose you could move the comment up under the existing one.

**Comment by live627** [comment]
will do it tomorrow

**Review by Sesquipedalian — APPROVED** [review]

**Pull request merged**